documentation/book/con-client-connection-tls.adoc

```diff
@@ -0,0 +1,47 @@
+// Module included in the following assemblies:
+//
+// <List assemblies here, each on a new line>
+
+// Base the file name and the ID on the module title. For example:
+// * file name: assembly-cluster-operator.adoc
+// * ID: [id='encryption-{context}']
+// * Title: = Encryption
+
+// The ID is used as an anchor for linking to the module. Avoid changing it after the module has been published to ensure existing links are not broken.
+
+include::common/attributes.adoc[]
+
+[id='client-connection-tls-{context}']
+= Kafka Client connections through TLS
+
+Encrypted communication between Kafka brokers and clients is provided through the `CLIENTTLS` listener on port 9093.
+
+NOTE: You can use the `CLIENT` listener on port 9092 for unencrypted communication with brokers.
+
+If a Kafka client wants to connect to the encrypted listener (`CLIENTTLS`) on port 9093, it needs to trust the cluster CA certificate in order to verify the broker certificate received during the SSL/TLS handshake.
```

Comment:
This seems like a procedure

Reply:
@scholzj , should a procedure be added here or a link to the upstream doc?